README.md

```diff
@@ -1915,8 +1915,11 @@ IEEE Int. Conf. Image Processing (ICIP), 2022. [PDF](https://arxiv.org/pdf/2202.
 - <a name="Gehrig23cvpr"></a>Gehrig, M., Scaramuzza, D.,  
 *[Recurrent Vision Transformers for Object Detection with Event Cameras](https://arxiv.org/abs/2212.05598)*,  
  IEEE Conf. Computer Vision and Pattern Recognition (CVPR), 2023. [PDF](https://rpg.ifi.uzh.ch/docs/CVPR23_mgehrig.pdf), [Code](https://github.com/uzh-rpg/RVT).
-- [El Shair et al., IEEE Access 2023](#Elshair23access).    
-*CSTR: A Compact Spatio-Temporal Representation for Event-Based Vision*.   
+- [El Shair et al., IEEE Access 2023](#Elshair23access).  
+*CSTR: A Compact Spatio-Temporal Representation for Event-Based Vision*.
+- <a name="Himmi24wacv"></a>Himmi, S., Parret, V., Chhatkuli , A., Van Gool, L.,  
+*[MS-EVS: Multispectral event-based vision for deep learning based face detection](https://arxiv.org/abs/2402.15584)*,  
```

Comment:
Thanks .Wrong arxiv link. Please revise.
I think it is better to use the official PDF link here, since you know it and it is available (open access). Then, no need to use [PDF] afterwards.

Reply:
Nice catch - stupid mistake 😅 

Thanks for the suggestion, I updated the link with the PDF 🙂